kitsune/search/migrations/0002_delete_record.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('search', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='Record',
+        ),
+    ]
```

Comment:
We don't care about previous records. It's way easier to delete the table and then create the new one rather than try to migrate between them.